dnf-behave-tests/features/append_options.feature

```diff
@@ -0,0 +1,286 @@
+Feature: DNF test of append option
+
+# The excludepks append option was choosen for testing.
+# Both paths "--exclude" and "--setopt=excludepkgs" are tested.
+
+Scenario: Test that excludes from config file are applied
+  Given I use repository "dnf-ci-fedora-updates"
+    And I do not set config file
+    And I create file "/etc/dnf/dnf.conf" with
+    """
+    [main]
+    exclude = lame, lz4
+    """
```

Comment:
Hey, I think you want to put these steps into `Background:` to avoid the repetition (they seem t obe the same for all the Scenarios, unless I missed some differences).